scripts/install

```diff
@@ -226,6 +226,12 @@ ynh_app_setting_set --app=$app --key=registration_shared_secret --value="$regist
 ynh_app_setting_set --app=$app --key=form_secret --value="$form_secret"
 ynh_print_ON
 
+# Create .well-known redirection for access by federation
+if yunohost --output-as plain domain list | grep -q "^$server_name$"
+then
+    cp ../conf/server_name.conf /etc/nginx/conf.d/${server_name}.d/server_name.conf
```

Comment:
Can you add checksum management for this file with the helper `ynh_store_file_checksum` as same as for the other config file.

You also need to replace `__SERVER_NAME__` and `__PORT__` here as same as for the other config files.

Reply:
I hope I did it good. I'm not so sure about replacing __SERVER_NAME__ ...